src/liballoc/collections/btree/map.rs

```diff
@@ -211,6 +211,51 @@ impl<K: Clone, V: Clone> Clone for BTreeMap<K, V> {
             clone_subtree(self.root.as_ref())
         }
     }
+
+    default fn clone_from(&mut self, other: &Self) {
```

Comment:
This should be negotiated via an internal trait rather than using `default fn` here.

Reply:
Should be fixed now. What kind of documentation do these internal traits need? Just a comment saying that everything external is going through the `Clone + Ord` path?

Reply:
Probably nothing; it's obvious enough from the context.